### DISCUSSION TITLE
I install follow document and when try test it get Error [ERR_REQUIRE_ESM]: require() of ES Module

### DISCUSSION THREAD

**Discussion by duongca** [spatie/lighthouse-php#15] [Q&A]
Could you help me check the issue with error log

[2023-02-10 05:11:26] development.ERROR: Could not run Lighthouse. Error : `E:\MyProjects\theseotool.site\vendor\spatie\lighthouse-php\bin\lighthouse.js:1
const lighthouse = require('lighthouse');
                   ^

Error [ERR_REQUIRE_ESM]: require() of ES Module E:\MyProjects\theseotool.site\node_modules\lighthouse\core\index.js from E:\MyProjects\theseotool.site\vendor\spatie\lighthouse-php\bin\lighthouse.js not supported.
Instead change the require of index.js in E:\MyProjects\theseotool.site\vendor\spatie\lighthouse-php\bin\lighthouse.js to a dynamic import() which is available in all CommonJS modules.
    at Object.<anonymous> (E:\MyProjects\theseotool.site\vendor\spatie\lighthouse-php\bin\lighthouse.js:1:20) {
  code: 'ERR_REQUIRE_ESM'
}

Node.js v18.12.1
` {"exception":"[object] (Spatie\\Lighthouse\\Exceptions\\CouldNotRunLighthouse(code: 0): Could not run Lighthouse. Error : `E:\\MyProjects\\theseotool.site\\vendor\\spatie\\lighthouse-php\\bin\\lighthouse.js:1

const lighthouse = require('lighthouse');

**Comment by Gerardlc** [comment]
same error, fresh laravel 9 install, using sail and adding the chromium-browser to de dockerfile

**Comment by bmartus** [comment]
It looks like lighthouse latest version (10) converted from CommonJS to ES modules which is a breaking change.

A quick way to get this to work as-is is to `npm require lighthouse@9.6.8` until lighthouse-php is updated to use the new version of lighthouse.

**Comment by justinbuhrmann1997** [comment]
It might be noteworthy to add something to the documentation about the lighthouse package version requirement.
Since this does not seem to work with lighthouse 10.* .

**Comment by freekmurze** [comment]
I'll investigate this soon, I'd like to be up to date with Lighthouse 10. 

If any of you already knows what needs to be changed, I'd accept a PR. 

If breaking changes are needed to support Lighthouse 10, I'd be willing to tag a new major version of the package.

**Reply by justinbuhrmann1997** [reply]
I have submitted a PR containing a fix for importing the lighthouse module. This should work both with lighthouse 10+ and versions prior to this.

**Reply by Pluuk** [reply]
The PR isn't approved yet. Is there any update on this?

**Reply by justinbuhrmann1997** [reply]
It has been merged already (https://github.com/spatie/lighthouse-php/pull/20)

**Reply by Pluuk** [reply]
Seems to still be a problem for some installations then, I'm running laravel/framework 10.16.1, spatie/lighthouse-php 1.0.5, lighthouse@10.4.0, vite@3.2.7 and the issue is still occuring for me. 

EDIT: Tested with Node 16, 18 and 20 and all are failing for me, so that can't be it.

**Reply by Pluuk** [reply]
The issue was with chrome-launcher, not with lighthouse. I've created #28 to fix this issue.
